system/jlib/jstats.cpp

```diff
@@ -1067,6 +1071,14 @@ const char * queryStatisticName(StatisticKind kind)
 }
 
 
+const char * queryStatisticDescription(StatisticKind kind)
+{
+    StatisticKind rawkind = (StatisticKind)(kind & StKindMask);
+    if (rawkind >= StKindNone && rawkind < StMax)
```

Comment:
result will be same I think either way, but should this be rawKind > StKindNone ?

Reply:
It is consistent with the other functions.